src/seedu/addressbook/data/person/NameTest.java

```diff
@@ -39,6 +39,8 @@ public void test() {
         assertTrue("Same name", name.isSimilar(createName("John Smith")));
         
         assertTrue("Same name with different cases", name.isSimilar(createName("joHn sMith")));
+        
```

Comment:
errr.. i see continuous updates to the tests. where is the method being developed?